pkg/cmd/deprecation/deprecated.go

```diff
@@ -8,84 +8,85 @@ import (
 	},
 	"init": {
-		replacement: "jx boot",
-		date:        "Jun 1 2020",
-		info: fmt.Sprintf("Please check %s for more details.",
+		Replacement: "jx boot",
+		Date:        "Jun 1 2020",
+		Info: fmt.Sprintf("Please check %s for more details.",
 			util.ColorStatus("https://jenkins-x.io/docs/getting-started/setup/boot/")),
 	},
 	"create post": {
-		date: "Feb 1 2020",
+		Date: "Feb 1 2020",
 	},
 	"create spring": {
-		replacement: "jx create project",
-		date:        "Mar 1 2020",
-		info: fmt.Sprintf("Please check %s for more details.",
+		Replacement: "jx create project",
+		Date:        "Mar 1 2020",
+		Info: fmt.Sprintf("Please check %s for more details.",
 			util.ColorStatus("https://jenkins-x.io/commands/jx_create_project/")),
 	},
 	"upgrade platform": {
-		replacement: "jx upgrade boot",
-		date:        "Jun 1 2020",
-		info: fmt.Sprintf("Please check %s for more details.",
+		Replacement: "jx upgrade boot",
+		Date:        "Jun 1 2020",
+		Info: fmt.Sprintf("Please check %s for more details.",
 			util.ColorStatus("https://jenkins-x.io/docs/getting-started/setup/boot/")),
 	},
 	"upgrade ingress": {
-		replacement: "jx boot",
-		date:        "Jun 1 2020",
-		info: fmt.Sprintf("Please check %s for more details.",
+		Replacement: "jx boot",
+		Date:        "Jun 1 2020",
+		Info: fmt.Sprintf("Please check %s for more details.",
 			util.ColorStatus("https://jenkins-x.io/docs/getting-started/setup/boot/")),
 	},
 	"get post": {
-		date: "Feb 1 2020",
+		Date: "Feb 1 2020",
 	},
 	"delete extension": {
-		date: "Feb 1 2020",
-		info: "This is now replaced by apps",
+		Date: "Feb 1 2020",
+		Info: "This is now replaced by apps",
 	},
 	"edit extensionsrepository": {
-		date: "Feb 1 2020",
-		info: "This is now replaced by apps",
+		Date: "Feb 1 2020",
+		Info: "This is now replaced by apps",
 	},
 	"step post": {
-		date: "Feb 1 2020",
+		Date: "Feb 1 2020",
 	},
 	"step nexus drop": {
-		date: "Feb 1 2020",
+		Date: "Feb 1 2020",
 	},
 	"step nexus release": {
-		date: "Feb 1 2020",
+		Date: "Feb 1 2020",
 	},
 	"create vault": {
-		date: "Sep 1 2020",
-		info: "This commands will have no replacement.",
+		Date: "Sep 1 2020",
+		Info: "This commands will have no replacement.",
 	},
 	"delete vault": {
-		date: "Sep 1 2020",
-		info: "This commands will have no replacement.",
+		Date: "Sep 1 2020",
+		Info: "This commands will have no replacement.",
 	},
 }
 
 // deprecateInfo keeps some deprecation details related to a command
-type deprecationInfo struct {
-	replacement string
-	date        string
-	info        string
+type DeprecationInfo struct {
+	Replacement string
+	Date        string
+	Info        string
 }
 
 // DeprecateCommands runs recursively over all commands and set the deprecation message
 // on every command defined the deprecated commands map.
 func DeprecateCommands(cmd *cobra.Command) {
 	path := commandPath(cmd)
-	if deprecation, ok := deprecatedCommands[path]; ok {
-		cmd.Deprecated = deprecationMessage(deprecation)
```

Comment:
As you can see, we are no longer marking the cmd as "Deprecated", but updating the LongExample.  This is not ideal as it won't display when the command is run, just when help is called